code-studio/src/js/progress.js

```diff
@@ -160,9 +160,11 @@ progress.renderStageProgress = function (stageData, progressData, clientProgress
 };
 
 progress.renderCourseProgress = function (scriptData) {
+  var teacherCourse = $('#landingpage').hasClass('teacher-course');
   var mountPoint = document.createElement('div');
   $('.user-stats-block').prepend(mountPoint);
   ReactDOM.render(React.createElement(CourseProgress, {
+    display: teacherCourse ? 'list' : 'dots',
```

Comment:
Nit: It'd be nice for `'list'` and `'dots'` to be constants exported by CourseProgress.